app/scaffold/scaffold.js

```diff
@@ -181,14 +195,19 @@ class Scaffold {
             if (!promise) {
                 return;
             }
+
+            DOM.alertBox("Attempting to decrypt...");
```

Comment:
This should just be "Decrypting..." Since the action is in process at the point the loading message displays.